api/src/main/java/io/grpc/LoadBalancer.java

```diff
@@ -682,6 +706,12 @@ public static PickResult withNoResult() {
       return NO_RESULT;
     }
 
+    /** Returns the authority override hostname if any. */
+    @Nullable
+    public String getAuthorityOverrideHostname() {
```

Comment:
Why is "hostname" is the method name? It basically never makes sense to say both authority and hostname.

Reply:
Done.